service/controller/v10/adapter/security_groups.go

```diff
@@ -5,6 +5,7 @@ import (
 	"github.com/giantswarm/microerror"
 
 	"github.com/giantswarm/aws-operator/service/controller/v10/key"
+	"strings"
```

Comment:
The imports are usually sorted in three different blocks. Stdlib, external dependencies, repo internal packages.